docs/UserGuide.md

```diff
@@ -67,12 +67,15 @@ Format: `list_student` <br>
 
 ### Locating student profile by name: `find_student`
 
-Finds students that matches the keyword given
+Finds students whose names contain any of the given keywords.
 
-Format: `find_student KEYWORD`
-* The search will be case-insensitive. e.g. searching “stonks” will match “STONKS”.
-* As long as the keyword matches the name of the student, it will be regarded as a match. For example, if a student’s name is “John Lee”, searching “John” will be sufficient.
+Format: `find_student KEYWORD [MORE_KEYWORDS]`
+* The search will be case-insensitive. e.g. searching `stonks` will match `STONKS`.
 * Only the student’s name will be searched.
+* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+* Only full words will be matched e.g. `Han` will not match `Hans`.
+* Students matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
```

Comment:
HAHA this one abit ocd but perhaps make all the points not end with a dot for uniformity, otherwise lgtm!

Reply:
Haha alright! Updated